js/util/is_char_in_unicode_block.js

```diff
@@ -0,0 +1,268 @@
+    // 'Latin Extended-B': (char) => char >= 0x0180 && char <= 0x024F,
+    // 'IPA Extensions': (char) => char >= 0x0250 && char <= 0x02AF,
+    // 'Spacing Modifier Letters': (char) => char >= 0x02B0 && char <= 0x02FF,
+    // 'Combining Diacritical Marks': (char) => char >= 0x0300 && char <= 0x036F,
+    // 'Greek and Coptic': (char) => char >= 0x0370 && char <= 0x03FF,
+    // 'Cyrillic': (char) => char >= 0x0400 && char <= 0x04FF,
+    // 'Cyrillic Supplement': (char) => char >= 0x0500 && char <= 0x052F,
+    // 'Armenian': (char) => char >= 0x0530 && char <= 0x058F,
+    // 'Hebrew': (char) => char >= 0x0590 && char <= 0x05FF,
+    // 'Arabic': (char) => char >= 0x0600 && char <= 0x06FF,
+    // 'Syriac': (char) => char >= 0x0700 && char <= 0x074F,
+    // 'Arabic Supplement': (char) => char >= 0x0750 && char <= 0x077F,
+    // 'Thaana': (char) => char >= 0x0780 && char <= 0x07BF,
+    // 'NKo': (char) => char >= 0x07C0 && char <= 0x07FF,
+    // 'Samaritan': (char) => char >= 0x0800 && char <= 0x083F,
+    // 'Mandaic': (char) => char >= 0x0840 && char <= 0x085F,
+    // 'Arabic Extended-A': (char) => char >= 0x08A0 && char <= 0x08FF,
+    // 'Devanagari': (char) => char >= 0x0900 && char <= 0x097F,
+    // 'Bengali': (char) => char >= 0x0980 && char <= 0x09FF,
+    // 'Gurmukhi': (char) => char >= 0x0A00 && char <= 0x0A7F,
+    // 'Gujarati': (char) => char >= 0x0A80 && char <= 0x0AFF,
+    // 'Oriya': (char) => char >= 0x0B00 && char <= 0x0B7F,
+    // 'Tamil': (char) => char >= 0x0B80 && char <= 0x0BFF,
+    // 'Telugu': (char) => char >= 0x0C00 && char <= 0x0C7F,
+    // 'Kannada': (char) => char >= 0x0C80 && char <= 0x0CFF,
+    // 'Malayalam': (char) => char >= 0x0D00 && char <= 0x0D7F,
+    // 'Sinhala': (char) => char >= 0x0D80 && char <= 0x0DFF,
+    // 'Thai': (char) => char >= 0x0E00 && char <= 0x0E7F,
+    // 'Lao': (char) => char >= 0x0E80 && char <= 0x0EFF,
+    // 'Tibetan': (char) => char >= 0x0F00 && char <= 0x0FFF,
+    // 'Myanmar': (char) => char >= 0x1000 && char <= 0x109F,
+    // 'Georgian': (char) => char >= 0x10A0 && char <= 0x10FF,
+    'Hangul Jamo': (char) => char >= 0x1100 && char <= 0x11FF,
+    // 'Ethiopic': (char) => char >= 0x1200 && char <= 0x137F,
+    // 'Ethiopic Supplement': (char) => char >= 0x1380 && char <= 0x139F,
+    // 'Cherokee': (char) => char >= 0x13A0 && char <= 0x13FF,
+    // 'Unified Canadian Aboriginal Syllabics': (char) => char >= 0x1400 && char <= 0x167F,
+    // 'Ogham': (char) => char >= 0x1680 && char <= 0x169F,
+    // 'Runic': (char) => char >= 0x16A0 && char <= 0x16FF,
+    // 'Tagalog': (char) => char >= 0x1700 && char <= 0x171F,
+    // 'Hanunoo': (char) => char >= 0x1720 && char <= 0x173F,
+    // 'Buhid': (char) => char >= 0x1740 && char <= 0x175F,
+    // 'Tagbanwa': (char) => char >= 0x1760 && char <= 0x177F,
+    // 'Khmer': (char) => char >= 0x1780 && char <= 0x17FF,
+    'Mongolian': (char) => char >= 0x1800 && char <= 0x18AF,
```

Comment:
Per https://github.com/mapbox/mapbox-gl-js/pull/3438#discussion_r84585841, we don’t want to rotate Mongolian vertically or give it ideographic breaking.